game/src/core/systems/HudSystem.java

```diff
@@ -0,0 +1,78 @@
+package core.systems;
+
+import com.badlogic.gdx.scenes.scene2d.Group;
+
+import core.Game;
+import core.System;
+import core.components.UIComponent;
+
+public final class HudSystem extends System {
+
+    public HudSystem() {
+        super(UIComponent.class);
+    }
+
+    /* for dungeon Issue #727
+        @Override
+        private void addEntity(Entity entity) {
+            stage.addActor(entity.getComponent(UIComponent.class).map(UIComponent.class::cast).map(UIComponent::getDialog).get());
+        }
+    */
+
+    /* for dungeon Issue #727
```

Comment:
auskommentierter code.

wollen wir das wirklich einchecken hier? sollte das nicht besser dann in dem betreffenden issue vorgeschlagen werden?